community/dbms/src/main/java/org/neo4j/commandline/dbms/ImportCommand.java

```diff
@@ -41,10 +41,15 @@
 import org.neo4j.kernel.impl.util.Validators;
 import org.neo4j.server.configuration.ConfigLoader;
 
+import static org.neo4j.csv.reader.Configuration.DEFAULT;
+import static org.neo4j.unsafe.impl.batchimport.Configuration.DEFAULT_MAX_MEMORY_PERCENT;
+import static org.neo4j.unsafe.impl.batchimport.input.csv.Configuration.COMMAS;
+
 public class ImportCommand implements AdminCommand
 {
     public static final String DEFAULT_REPORT_FILE_NAME = "import.report";
     private static final String[] allowedModes = {"database", "csv"};
+    private static final String[] allowedDelimiterCharacters = {",", "TAB", ";", "|", ":", "#"};
```

Comment:
I thought it'd be best to have a strict set of delimiters because there are times other special characters are used in a different context. Ex: If you specify `\` as delimiter it would conflict because we are turning on` --legacy-style-quoting=true` by default.

Having said this, I have just now verified that the allowedDelimiterCharacters is of no use because there are no validations done on verifying on delimiter character.